src/icons/circle-check-outlined.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I suggest to import it as a square to avoid "egg shape" when providing the same width and height (scaling). Ask Tsvetan about help.